src/main/scala/io/github/pauljamescleary/petstore/infrastructure/endpoint/alias.scala

```diff
@@ -0,0 +1,11 @@
+package io.github.pauljamescleary.petstore
+package infrastructure.endpoint
```

Comment:
remove `endpoint`, see below, this package object should be defined as endpoint so its contents are available within the endpoint package.